config/prow/config.yaml

```diff
@@ -70,7 +70,7 @@ deck:
     - lens:
         name: junit
       required_files:
-        - ^artifacts/junit.*\.xml$
+        - ^artifacts(/.*/|/)junit.*\.xml$ # https://regex101.com/r/vCSegS/1
```

Comment:
does this match testgrid? /shrug

Reply:
yeah testgrid has recursive listing for junit files.

Reply:
i mean will this match the same set of files?